src/styles/Story.module.css

```
@@ -0,0 +1,122 @@
+.storyWrapper {
+  font-family: "Verdana", sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Verdana" (font-family-name-quotes)

(font-family-name-quotes)